src/Razor/src/Microsoft.CodeAnalysis.Razor/ComponentTagHelperDescriptorProvider.cs

```diff
@@ -333,6 +317,7 @@ private void CreateContextParameter(TagHelperDescriptorBuilder builder, string c
         // a dictionary keyed on property name.
         //
         // We consider parameters to be defined by properties satisfying all of the following:
+        // - are public
```

Comment:
Requiring public wasn't initially mentioned in the component visibility issue but it felt "more right" to me in an effort to default to being more restrictive for RTM. Let me know if you feel differently.

Reply:
sure